Issue #2690 - clarify websocket message ordering in documentation (#7097)
eclipse/jetty.project: documentation/jetty-documentation/src/main/asciidoc/programming-guide/websocket.adoc

Applications interested in the error event receive a `Throwable` that represent the error.
* The _message_ event.
The message event is emitted when a WebSocket message is received.

> Only a single thread will be delivering a message event at any one time, and the next message event will not be triggered until the previous one has exited its onMessage method.

Reviewer: ```suggestion
Only one thread at a time will be delivering a message event to the `@OnMessage` method; the next message event will not be delivered until the previous call to the `@OnMessage` method has exited.
```

Author: I didn't want to use that specific `@OnMessage` annotation because this that annotation is specific to `javax.websocket`. This is talking about a more general message event which could be a method from a listener class or the `@OnWebSocketMessage` annotation from Jetty API.
